operator/constants/constants.go

```diff
@@ -17,3 +17,26 @@ const (
 
 	FirstPortNumber = 9000
 )
+
+const (
+	ControllerName = "seldon-core-controller"
```

Comment:
Was the name not seldon-controller-manager? Or is that for different resource?

Reply:
yes - worth being consistent - will change